dts/arm/st/f7/stm32f7.dtsi

```diff
@@ -652,6 +659,12 @@
 		#phy-cells = <0>;
 		label = "OTGFS_PHY";
 	};
+
+	sdram0: memory@c0000000 {
```

Comment:
Since SDRAM is mounted on board, I'd expect this to be declared in board.dts

Reply:
I totally agree! Will fix that.